documentation/src/main/asciidoc/user_guide/clustered_lock.adoc

```diff
@@ -0,0 +1,200 @@
+== Clustered Lock
+
+Clustered lock is a new building block introduced in Infinispan 9.2
+It is a lock distributed and shared among all nodes in the Infinispan cluster and currently provides a way to execute
+code that will be synchronized between the nodes in a given cluster.
+
+=== Installation
+
+In order to start using the clustered locks, you needs to add the dependency in your Maven `pom.xml` file:
+
+.pom.xml
+[source,xml]
+----
+<dependency>
+   <groupId>org.infinispan</groupId>
+   <artifactId>infinispan-clustered-lock</artifactId>
+   <version>...</version> <!-- 9.2.0.Final or higher -->
```

Comment:
@tristantarrant Maybe we should have a placeholder like `{infinispanversion}` but with the full version?

Reply:
Indeed. @karesti please add a `<fullinfinispanversion>${project.version}</fullinfinispanversion>` to `documentation/pom.xml` in the `<attributes>` section so that we can substitute it correctly when docs are generated

Reply:
but I don't want this value to be replaced each time. This is something from 9.2.0.Final. I did the same thing as in counters where it is said <!-- 9.0.0.Final or higher -->
I will do this in a separate PR you want